src/path_converters.h

```diff
@@ -363,10 +366,30 @@ class PathClipper : public EmbeddedQueue<3>
                     goto exit_loop;
 
                 case agg::path_cmd_move_to:
+
+		    // was the last command a move to (and we have
+		    // seen at least one command ?
+		    // if so, shove it in the queue if in clip box
+		    if (m_moveto && m_has_init &&
+			m_lastX >= m_cliprect.x1 &&
+			m_lastX <= m_cliprect.x2 &&
+			m_lastY >= m_cliprect.y1 &&
+			m_lastY <= m_cliprect.y2) {
+			// push the last moveto onto the queue
+                        queue_push(agg::path_cmd_move_to, m_lastX, m_lastY);
+			// flag that we need to emit it
+			emit_moveto = true;
+                    }
+		    // update the internal state for this move to
                     m_initX = m_lastX = *x;
                     m_initY = m_lastY = *y;
                     m_has_init = true;
                     m_moveto = true;
+		    // if the last command was move to exit the loop to emit the code
```

Comment:
I mean, this is pretty clearly what it does already, so it's kind of a redundant comment. Would be better as a why instead of a what.